src/utils/self_test/SConscript

```diff
@@ -13,19 +13,18 @@ SELF_TEST = 'self_test.c'
 def scons():
     """scons function"""
 
-    Import('env', 'prereqs', 'cart_lib', 'gurt_lib')
-
+    Import('env', 'prereqs', 'cart_lib', 'gurt_lib', 'cart_utils_lib')
     tenv = env.Clone()
 
     prereqs.require(tenv, 'pmdk')
     tenv.AppendUnique(CPPPATH=['#/src/cart', '#/src/tests/ftest/cart'])
-    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'm'])
+    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'm', 'cart_utils'])
```

Comment:
I think you can also remove both of these CPPPATH entries for same reason as LIBPATH below